### PR TITLE
should let us add new column property: function that return css clas…

### PR DESCRIPTION
**Pull request by HeorhiSakalouski**
simple way to highlight grid cells using row data

### CODE REVIEW

File: bs-config.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
     "routes": {
         "/": ""
     }
-  }
-}
\ No newline at end of file
+  },
```
Comment (branimirborisov) [inline bs-config.json]: remove this

File: demo/styles.css [hunk 2]
```diff
@@ -152,4 +152,12 @@ nav a.router-link-active {
 }
 .mb-md {
   margin-bottom: 15px;
-}
\ No newline at end of file
+}
+
+.cell_red {
```
Comment (branimirborisov) [inline demo/styles.css]: lets keep kebab-case style convention for class name, so rename to .cell-red, .cell-green

File: demo/demo.component.ts [hunk 3]
```diff
@@ -153,4 +153,12 @@ export class DemoComponent implements OnInit, AfterViewInit {
   onGridEvent(event: GridEvent) {
     console.log(event);
   }
+
+  getCellCssClass(data: any) {
```
Comment (branimirborisov) [inline demo/demo.component.ts]: add return value to method

File: package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "clean": "gulp clean",
     "start": "tsc && concurrently \"tsc -w\" \"lite-server\" ",
-    "e2e": "tsc && concurrently \"http-server -s\" \"protractor protractor.config.js\" --kill-others --success first",
+    "e2e": "tsc && concurrently \"tsc -w\" \"lite-server\" \"sleep 25 && protractor protractor.config.js\" --kill-others --success first",
```
Comment (branimirborisov) [inline package.json]: remove this change

File: protractor.config.js [hunk 5]
```diff
@@ -16,6 +16,8 @@ var _ = require('lodash');
 
 
 exports.config = {
+  allScriptsTimeout: 1100000,
```
Comment (branimirborisov) [inline protractor.config.js]: revert all changes in this file

File: src/grid-column.component.ts [hunk 6]
```diff
@@ -86,3 +92,7 @@ export class GridColumnComponent implements OnInit {
       return _.get(data, columnName) as string;
   }
 }
+
+export interface IStyleCallback {
```
Comment (branimirborisov) [inline src/grid-column.component.ts]: Lets not use "I" convention for interface names. Also lets include Cell in the name, so CellStyleCallback?
We also have similar interface declared in grid-options.ts called RowStyleCallback, another option would be to combine those and move them to separate file with just StyleCallback name and share same interface.

File: src/grid-column.component.ts [hunk 7]
```diff
@@ -42,6 +42,7 @@ export class GridColumnComponent implements OnInit {
   @Input() items: any;
   @Input() textField: string;
   @Input() valueField: string;
+  @Input() cellStyleCallback: IStyleCallback;
```
Comment (branimirborisov) [inline src/grid-column.component.ts]: lets keep alphabetical order

**Comment by branimirborisov** [comment]
@HeorhiSakalouski I have fixed already tests in master branch so you can clean up your changes related to e2e test configuration

**Comment by HeorhiSakalouski** [comment]
@branimirborisov  i'v just tried to satisfy your issues, recheck it please

**Review by branimirborisov — APPROVED** [review]

**Comment by branimirborisov** [comment]
@HeorhiSakalouski looks good, thanks!

**Pull request merged**